functionary/functionary/settings/base/constance_.py

```diff
@@ -0,0 +1,67 @@
+CONSTANCE_BACKEND = "constance.backends.database.DatabaseBackend"
+CONSTANCE_DATABASE_PREFIX = "constance:functionary:"
+CONSTANCE_ADDITIONAL_FIELDS = {
+    "config_field": ["django.forms.fields.JSONField", {}],
+}
+CONSTANCE_CONFIG = {
+    "SOCIALACCOUNT_PROVIDERS": (
+        {
+            "github": {},
+            "gitlab": {},
+            "keycloak": {},
+        },
+        "SocialAccount Providers Configuration",
+        "config_field",
+    ),
+    "S3_HOST": ("localhost", "Server Hostname", str),
+    "S3_PORT": (9000, "Port", int),
+    "S3_REGION": ("", "Region", str),
+    "S3_SECURE": (False, "Require Secure Access", bool),
+    "S3_ACCESS_KEY": ("ACCESS_KEY", "ROOTUSER", str),
+    "S3_SECRET_KEY": ("SECRET_KEY", "CHANGEME123", str),
```

Comment:
The order of these settings appears to be (default value, name, type), meaning "CHANGEME123" is what is being shown as the setting name in the admin pages.  I'm assuming that's not the intention.